libvips/foreign/radiance.c

```diff
@@ -546,7 +546,7 @@ scanline_read_old( VipsSbuf *sbuf, COLR *scanline, int width )
 			scanline[0][BLU] == 1 ) {
 			int i;
```

Comment:
Should i be a guint? Otherwise 255 << 24 will make a huge -ve number and break the loop.

I don't think it's dangerous, but it would be a decode error.

Reply:
Ah, I overlooked that one. Commit https://github.com/libvips/libvips/commit/aac17486152b90de3f5d22cca1b30de1be6333fb fixes this. It is now able to decode `thumbnail_fuzzer-5111890150424576` without any problems.